src/Microsoft.Health.Dicom.Functions.UnitTests/Update/UpdateDurableFunctionTests.cs

```diff
@@ -20,6 +21,7 @@
 using NSubstitute;
 using OpenTelemetry;
 using OpenTelemetry.Metrics;
+using Metric = OpenTelemetry.Metrics.Metric;
```

Comment:
What type does this conflict with? There's something called `Metric` in `Microsoft.ApplicationInsights`?

Reply:
Yes, there is Metric in Application Insights